ios/Classes/BetterPlayer.h

```diff
@@ -66,9 +66,9 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)setPipSeekButtonsHidden:(BOOL) isHidden;
 - (void)setIsDisplayPipButtons:(BOOL) isDisplay;
 - (void)setIsPremiumBannerDisplay:(BOOL) isDisplay;
-- (void)setIsPreparingDatasource:(BOOL)isPreparingDatasource;
-- (void)showBlackCoverView;
-- (void)hideBlackCoverView;
+- (void)setIsPipMode:(BOOL) isPipMode;
```

Comment:
I think, `setIsPipMode` is only called from this class.
Could we remove this line from this header file?

Reply:
@noboru-i 
Sure.